app/components/project-geometries/modes/draw/feature-label-form.js

```diff
@@ -18,13 +18,50 @@ export default class FeatureLabelFormComponent extends Component {
   @argument
   options=null;
 
+  isBold = false;
+
+  isLarge = false;
+
+  @observes('selectedFeature')
+  selectedFeatureUpdated() {
+    this.set('isBold', false);
+    this.set('isLarge', false);
+  }
```

Comment:
Not sure that observes is the right choice here. `isBold` and `isLarge` are conceptually able to be computed downstream of ... `selectedFeature` being mutated? What about making `isBold` and `isLarge` into computeds themselves? 

There is also `this.notifyPropertyChange`. 

Lastly, we have a `updateSelectedFeature` - what if `updateSelectedFeature` was kept as a public argument, but we create a private method which internally calls `updateSelectedFeature` while also triggering necessary updates?

Or we could just keep an observer in the interest of time.